scala/scala.bzl

```diff
@@ -786,6 +803,8 @@ def scala_repositories():
 
   native.bind(name = "io_bazel_rules_scala/dependency/scalatest/scalatest", actual = "@scalatest//jar")
 
+# With providers changes, this is completely deprecatable once bazel release includes
+# fixes to java rules using JavaProvider
```

Comment:
can we change the implementation of this to just do the naive thing? We could make a `java_library` target that has the given exports and runtime_deps?